apps/frontend/components/field/referee/inspection-timer.tsx

```diff
@@ -0,0 +1,64 @@
+import { useMemo } from 'react';
+import dayjs from 'dayjs';
+import { Paper, Typography, Box } from '@mui/material';
+import Countdown from '../../general/countdown';
+import { useTime } from '../../../hooks/use-time';
+
+interface InspectionTimerProps {
+  startTime: string;
+}
+
+const INSPECTION_LENGTH = 90; // 1.5 minutes in seconds
```

Comment:
Move this to `constants.ts`, and name it INSPECTION_TIMER_LENGTH. Also create a const called SHOW_INSPECTION_TIMER which allows us to disable this feature.